frontend/src/main.tsx

```diff
@@ -1,11 +1,14 @@
+import { Provider } from "jotai";
```

Comment:
```suggestion
import { Provider as JotaiProvider } from "jotai";
```
으로 바꿔주실래요? 이름이 너무 generic해서.. 아니면
```
import jotai from "jotai";
```
로 하고 `jotai.Provider` 로 쓰거나요